src/tests/test_meta_blk_mgr.cpp

```diff
@@ -134,7 +137,7 @@ class VMetaBlkMgrTest : public ::testing::Test {
         return sec.count();
     }
 
-    [[nodiscard]] bool keep_running() {
+        [[nodiscard]] bool keep_running() {
```

Comment:
I think there is some misalignment in spaces. Existing change is correct. Perhaps format_on_save not enabled?

Reply:
my clang format plugin seems to be always get fooled with this keyword "[[nodiscard]]", let me fix it.